3.Linear_Modeling/1a.fit_linear_model_1beta.ipynb

```diff
@@ -0,0 +1,272 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #19.        X = df.loc[:, model_covariates]
</pre><p>I would recommend standardizing input feature values before training model. here's a [source](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.StandardScaler.html)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>@axiomcura turns out the data was already standardized via pycytominer. Mean of 0 and stdev of 1</p>